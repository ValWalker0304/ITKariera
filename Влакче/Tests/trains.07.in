Add 55924 StaraZagora F 9
Add 13674 Podkova P 1
Add 89478 Svilengrad F 15
Add 71333 Shumen F 21
Add 79282 Svilengrad P 3
Add 86488 GOryahovitsa F 29
Add 15366 Karnobat F 32
Next
Travel
Add 56579 StaraZagora F 23
Travel
Travel
Next
Add 63651 Targovishte F 39
Add 74876 Podkova F 29
Travel
Next
Travel
Next
Travel
Add 70963 Sliven F 32
Next
Next
Add 28478 StaraZagora F 1
Next
Add 70667 Sliven F 33
Add 78340 Sofia F 23
Travel
Next
Add 87428 Montana F 39
Travel
Next
Next
Add 76496 Zimnitsa F 35
Next
Travel
Add 44468 Ruse P 9
Next
Travel
Next
Travel
Next
Add 17732 Svilengrad F 33
Add 84935 Shumen P 7
Next
Travel
Travel
Next
Travel
Travel
Travel
Travel
Travel
Travel
Add 19556 Sindel P 5
Next
Add 89207 StaraZagora P 8
End