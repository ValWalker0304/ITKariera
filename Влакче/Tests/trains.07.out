15366 Karnobat F 32
15366 Karnobat F 32
56579 StaraZagora F 23
86488 GOryahovitsa F 29
71333 Shumen F 21
74876 Podkova F 29
63651 Targovishte F 39
63651 Targovishte F 39
71333 Shumen F 21
71333 Shumen F 21
70963 Sliven F 32
70963 Sliven F 32
79282 Svilengrad P 3
78340 Sofia F 23
70667 Sliven F 33
87428 Montana F 39
70667 Sliven F 33
70667 Sliven F 33
76496 Zimnitsa F 35
76496 Zimnitsa F 35
70667 Sliven F 33
70667 Sliven F 33
44468 Ruse P 9
44468 Ruse P 9
79282 Svilengrad P 3
17732 Svilengrad F 33
17732 Svilengrad F 33
84935 Shumen P 7
79282 Svilengrad P 3
79282 Svilengrad P 3
13674 Podkova P 1
28478 StaraZagora F 1
70963 Sliven F 32
89478 Svilengrad F 15
55924 StaraZagora F 9
19556 Sindel P 5
